Code/GraphMol/MolStandardize/Tautomer.cpp

```diff
@@ -202,6 +201,21 @@ std::vector<ROMOL_SPTR> TautomerEnumerator::enumerate(
   PRECONDITION(!modifiedBonds || modifiedBonds->size() >= mol.getNumBonds(),
                "bitset too small");
```

Comment:
Do you need a precondition to check it any bit is set?

Reply:
I thought about that, but since it doesn't actually cause a problem in the code if the user ends up calling this with something that has values pre-set, it didn't feel like a PRECONDITION violation (which is more for "something went badly wrong in the code").
We could theoretically output a warning, but I'm not sure it's worth it. What do you think?

Reply:
Works for me.  Caveat emptor.